server.go

```diff
@@ -30,10 +30,12 @@ type server struct {
 
 	accountsRepository models.AccountsRepository
 	groupsRepository   models.GroupsRepository
+	tchatsRepository   models.ConversationsRepository
```

Comment:
Same here, let's keep the naming consistent and use `conversations` everywhere instead of `tchat`.